Data/IDayOffRequestRepository.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using stunning_robot_HR.Models;
+
+namespace stunning_robot_HR.Data
+{
+    public interface IDayOffRequestRepository
```

Comment:
A good looking interface. Something to refer back to as you create new repositories.